lib/core/icon/icon.component.stories.ts

```diff
@@ -0,0 +1,65 @@
+/*!
+ * @license
+ * Copyright 2022 Alfresco Software, Ltd.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { Meta, moduleMetadata, Story } from '@storybook/angular';
+import { CoreStoryModule } from '../testing/core.story.module';
+import { IconComponent } from './icon.component';
+import { IconModule } from './icon.module';
+
+export default {
+    component: IconComponent,
+    title: 'Core/Components/Icon',
+    decorators: [
+        moduleMetadata({
+            imports: [CoreStoryModule, IconModule]
+        })
+    ],
+    argTypes: {
+        color: {
+            control: 'radio',
+            options: ['primary', 'accent', 'warn'],
+            description: 'icon color',
+            defaultValue: 'primary',
```

Comment:
do we need default value also here, when we have it in the line 39? 
https://github.com/Alfresco/alfresco-ng2-components/pull/7728/files#diff-a5937f7fd673b8778ce28fbb69d6bf3b87fb152492410ca53a0c27372ade63b5R39

Reply:
Yes, because line 36 marks the radio button. Line 39 gives a nice description in the documentation section 😄
![icon-story](https://user-images.githubusercontent.com/64163335/181272392-07211a34-c80f-45a8-91c4-5aaf037c7e26.png)

Reply:
thanks 👍